--- Cura-2.3.1/cura/CuraApplication.py.orig
+++ Cura-2.3.1/cura/CuraApplication.py
@@ -429,7 +429,7 @@
         self._plugin_registry.addType("profile_writer", self._addProfileWriter)
         self._plugin_registry.addPluginLocation(os.path.join(QtApplication.getInstallPrefix(), "lib", "cura"))
         if not hasattr(sys, "frozen"):
-            self._plugin_registry.addPluginLocation(os.path.join(os.path.abspath(os.path.dirname(__file__)), "..", "plugins"))
+            self._plugin_registry.addPluginLocation(os.path.join(os.path.abspath(os.path.dirname(__file__)), "plugins"))
             self._plugin_registry.loadPlugin("ConsoleLogger")
             self._plugin_registry.loadPlugin("CuraEngineBackend")
 
